projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -56,8 +60,12 @@ export class DotFormComponent {
     onValueChange(event: CustomEvent): void {
         const { tagName } = event.target as HTMLElement;
         const { name, value } = event.detail;
-        const transform = fieldParamsConversionToBE[tagName];
-        this.value[name] = transform ? transform(value) : value;
+        const process = fieldCustomProcess[tagName];
+        if (tagName === 'DOT-BINARY-FILE') {
```

Comment:
Why you didn't use the map `fieldCustomProcess` for extra process of fileds here?

Reply:
because in the case of `BINARY -FILE` we need to wait for a promise and then update values in the `dot-Form` context